src/com/company/Sudoku.java

```diff
@@ -0,0 +1,380 @@
+package com.company;
+
+import javax.swing.*;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class Sudoku {
+    public Sudoku(Scanner gameScan) {
+        this.gameScan = gameScan;
+    }
+
+    private  final Scanner gameScan;
+
+    public  boolean startup() {
+
+        System.out.println("Witamy w Sudoku Zugaj Edition V1\nBy rozpocząć rozgrywkę wpisz:  Sudoku");
+        String x= gameScan.next();
+        return x.equals("Sudoku");
+    }
+
+    public boolean validatePion(int[][] board){
+        for(int z=0;z<=8;z++){
+            for(int i=0;i<=8;i++) {
+                for(int j=0;j<=8;j++){
+                    if(i!=j){
```

Comment:
I, j, z 🤯 co odpowiada za co, nie mam pojęcia i nie chce wiedzieć
Zamiast tego mogłoby być po prostu row, column, nie wiem czym jest z to nie powiem